.yamllint

```diff
@@ -0,0 +1,9 @@
+---
+
+extends: default
+
+rules:
```

Comment:
I still see a few annotations like `[comments] too few spaces before comment` and `[comments-indentation] comment not indented like content` still

We have some other rules in the IREE repo that could help there:
https://github.com/iree-org/iree/blob/2a3a2f721b0353d7fc11ba23cbfde8fa4a875e7a/.yamllint.yml#L19-L24